acir/src/circuit/opcodes/black_box_function_call.rs

```diff
@@ -76,6 +76,16 @@ pub enum BlackBoxFuncCall {
         inputs: Vec<FunctionInput>,
         outputs: Vec<Witness>,
     },
+    VerifyProof {
+        key: Vec<FunctionInput>,
+        proof: Vec<FunctionInput>,
+        public_inputs: Vec<FunctionInput>,
+        key_hash: FunctionInput,
+        input_aggregation_object: Vec<FunctionInput>,
+        // This is the recursive verification output aggregation object.
+        // The name `outputs` was kept to simplify code reuse with the other BlackBoxFuncCall's
```

Comment:
I think changing this to output_aggregation_object would be better especially since we no longer have a homogenous API for the black box functions

Reply:
I will change it to the specific name

Reply:
Changed